src/engine/routing_algorithms/tile_turns.cpp

```diff
@@ -41,8 +43,12 @@ getTileTurns(const datafacade::ContiguousInternalMemoryDataFacade<ch::Algorithm>
         return facade.GetGeometryIndex(edge.forward_segment_id.id).id;
     };
 
-    // Build an adjacency list for all the road segments visible in
-    // the tile
+    // To build a tile, we can only rely on the r-tree to quickly find all data visible within the
+    // tile
```

Comment:
Comment formatting is a bit ugly here, looks like clang-format auto-wrapped.  Can you clean this up?